apps/medon-fe/src/assets/ReactComponentAssets/ChangeProfileAvatar.tsx

```diff
@@ -0,0 +1,3 @@
+import { ReactComponent as ChangeProfilePictureSvg } from '../editPhotoIcon.svg';
+
+export default ChangeProfilePictureSvg;
```

Comment:
what is the reason to have this file?

Reply:
I'm transforming the svg into a react component and storing it inside the assets folder, but in a exclusive folder to deal only with that called "ReactComponentAssets". Is there a better way to deal with it? Thanks in advance